kernel/include/rendering.h

```diff
@@ -37,6 +37,14 @@ void put_binary_len(uint64_t value, uint64_t x, uint64_t y, uint8_t bits);
 void put_binary_32(uint64_t value, uint64_t x, uint64_t y);
 void put_binary_64(uint64_t value, uint64_t x, uint64_t y);
 
+// Put value in decimal at character coordinates
+// returns the number of characters written (including '-')
+uint8_t put_int(int64_t value, uint64_t x, uint64_t y);
```

Comment:
why is return of put_int a u8 when put_hex and put_binary u64?

Reply:
put_int will never print more than about 20 characters, the capacity of a u64 is therefore unnecessary. put_hex should also be a uin8_t IMO. In reality it doesn't really matter and we could use u64 instead, although it should probably be consistent.

Reply:
I'll change it to use size_t instead, for functions added in this PR and then make a separate commit to main where I fix it for the other put_ functions

Reply:
I'll just use uint64_t for consistency.